blocks/library/button/index.js

```diff
@@ -74,7 +74,7 @@ registerBlock( 'core/button', {
 			<span className="blocks-button" title={ title }>
 				<Editable
 					tagName="span"
-					placeholder={ wp.i18n.__( 'Write some text…' ) }
+					placeholder={ wp.i18n.__( 'Write label…' ) }
```

Comment:
I can —  but I'm wondering —  is there a different way? It kind of feels wrong to add a min-width to the button as a whole, just so the placeholder text will be visible. 

<img width="383" alt="screen shot 2017-05-11 at 12 23 58" src="https://cloud.githubusercontent.com/assets/1204802/25944758/bb806528-3644-11e7-8d95-cd18d537f384.png">

Especially when this gets translated to German which is 33% longer, the  min width would have to be adjusted again. Is there a way the placeholder can provide the min width?